docs/running-on-kubernetes.md

```diff
@@ -798,6 +798,38 @@ from the other deployment modes. See the [configuration page](configuration.html
     the Driver process. The user can specify multiple of these to set multiple environment variables.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.client.watch.reconnectIntervalInMs</code></td>
+  <td><code>1s</code></td>
+  <td>
+    Spark kubernetes client watch reconnectInterval, which is relevant to
+    <code>KUBERNETES_WATCH_RECONNECT_INTERVAL_SYSTEM_PROPERTY</code> in io.fabric8.kubernetes.client.Config.
```

Comment:
Use semantic meanings here as opposed to meanings specific to the code implementation. For example, this documentation becomes incorrect if we end up not using the Fabric8 Kubernetes client. The other settings provide some examples that this can be based on.

Reply:
Done